source/kraken/fill_disruption_from_database.h

```diff
@@ -102,9 +101,9 @@ struct DisruptionDatabaseReader {
             message = nullptr;
             channel = nullptr;
             pt_object = nullptr;
```

Comment:
don't need to `nullptr` [property](https://github.com/CanalTP/navitia/pull/2848/files#diff-6e63c97d3884a53ade05aea2c2d061c0R70) too ?

Reply:
This part of code is not modified in this PR.